src/applications/disability-benefits/all-claims/pages/unemployabilityFormUpload.js

```diff
@@ -0,0 +1,61 @@
+import fileUploadUI from 'us-forms-system/lib/js/definitions/file';
+import environment from '../../../../platform/utilities/environment';
+import { FIFTY_MB } from '../constants';
+import { DocumentDescription } from '../content/uploadFormDocuments';
+
+export const uiSchema = {
+  'ui:title': 'Upload VA Form 21-8940',
+  'ui:description': DocumentDescription,
+  form8940Upload: fileUploadUI('', {
+    itemDescription: 'Document Type (VA Form 21-8940)',
+    hideLabelText: true,
+    fileUploadUrl: `${environment.API_URL}/v0/upload_supporting_evidence`,
+    fileTypes: ['pdf', 'jpg', 'jpeg', 'png'],
+    maxSize: FIFTY_MB,
+    createPayload: file => {
+      const payload = new FormData();
+      payload.append('supporting_evidence_attachment[file_data]', file);
+
+      return payload;
+    },
+    parseResponse: (response, file) => ({
+      name: file.name,
+      confirmationCode: response.data.attributes.guid,
+    }),
+    // this is the uiSchema passed to FileField for the attachmentId schema
+    // FileField requires this name be used
+    attachmentSchema: {
```

Comment:
Since y'all don't need a second user-input field (`attachmentId` in some of the other 526 upload UIs) in addition to the name field, you don't need this property either.

Reply:
The fileUpload UI schema generator is pretty confusing :(